### PR TITLE
Add script for generate FIM events

### PR DESCRIPTION
**Pull request by juliamagan**
# Description

Added two scripts:
- create_files.py: Create testing file before running the test
- generate_fim_events: Modify files and registry to generate FIM events

### CODE REVIEW

File: deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    for _ in range(n_events):
+        if sys.platform == 'win32':
+            random_string = ''.join(random.choice(string.ascii_letters) for _ in range(10))
+            for n_registry in range(1, len(test_files)+1):
+                key_h = win32api.RegOpenKeyEx(registry_parser[KEY], f'{testreg}{n_registry}', 0, KEY_ALL_ACCESS | KEY_WOW64_64KEY)
+                modify_registry_value(key_h, reg_value, REG_SZ, random_string)
+                generated_events += 1
+        else:
+            random_string = ''.join(random.choice(string.ascii_letters) for _ in range(file_size))
+            print(random_string)
```
Comment (Rebits) [inline deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py:123]:
```suggestion
```

File: deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                    f.write(random_string)
+                generated_events += 1
+
+        random_string = ''.join(random.choice(string.ascii_letters) for _ in range(file_size))
+        for filename in test_files[0:remain_events]:
+            with open(os.path.join(monitored_directory, filename), 'w+') as f:
+                f.write(random_string)
+            generated_events += 1
+
+    print(f'Generated {generated_events} events')
```
Comment (Rebits) [inline deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py:135]:
```suggestion
```

**Review by Rebits — CHANGES_REQUESTED** [review]
GJ, some minor changes are requested

File: deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py (hunk 1 above)
Comment (juliamagan) [reply deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py]: Done in [804fb74](https://github.com/wazuh/wazuh-qa/pull/4972/commits/804fb74d4e757a727d519e5e32c8cdf833567aa0)

File: deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py (hunk 2 above)
Comment (juliamagan) [reply deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py]: Done in 804fb74

**Pull request merged**